packages/fast-web-utilities/src/html.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { convertStylePropertyPixelsToNumber, getClientRectWithMargin } from "./html";
+
+describe("getClientRectWithMargin", () => {
+    const mockWidth: number = 120;
+    const mockHeight: number = 120;
+    const mockRect: any = {
```

Comment:
Instead of any can this be `ClientRectList` or `DOMRectList`?

Reply:
I'll check if we can type it and have it take. I think we may be able to cast it worst case scenario.